package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "orderentry",
```

Comment:
I'm wondering if one of these chnanges (which assumedly happened after a npm install) were the source of the failing testing... one thing I don't think the rest of us tried was running an npm  install and then running the test... will be interesting to see if that test fails on Bamboo after this merges... if it does, I think it would likely mean one of two things:

* There was a bug in one of the dependency upgrades that introduced an error, or
* There was a change in one of the dependencies that exposed a bug in our code or our tests...